src/module.cpp

```diff
@@ -210,6 +210,29 @@ static void lSetCodeModel(llvm::Module *module) {
     }
 }
 
+/** We need to set the correct value inside "PIC Level" metadata as it can be
+    used by some targets to generate code in the different way. Unlike code
+    model, it is not passed to the TargetMachine, so it is the only place for
+    codegen to access the correct value. */
+static void lSetPicLevel(llvm::Module *module) {
```

Comment:
I'd keep PIC uppercase but it's up to you.
```suggestion
static void lSetPICLevel(llvm::Module *module) {
```

Reply:
+1 for PIC

Reply:
Done